src/keygen/KeygenOptions.hs

```diff
@@ -0,0 +1,93 @@
+{-# LANGUAGE ApplicativeDo #-}
+
+module KeygenOptions
+       ( KeygenOptions (..)
+       , AvvmStakeOptions (..)
+       , TestStakeOptions (..)
+       , optsInfo
+       ) where
+
+import           Options.Applicative (Parser, ParserInfo, auto, fullDesc, help, helper,
+                                      info, long, metavar, option, option, progDesc,
+                                      short, strOption, switch, value)
+import           Universum
+
+data KeygenOptions = KO
```

Comment:
The naming is really bad b/c is obscure. One can't figure out what is it related to from constructor name => bad.